### PR TITLE
Add app function

### PR DESCRIPTION
**Pull request by denis-jullien**
Include already developped functions in /app : 
authentification with fastapi-users 
Book notice query from Google books & sudoc
#31

### CODE REVIEW

File: backend/models.py [hunk 1]
```diff
@@ -51,14 +76,14 @@ class BookCreate(BookBase):
...
-    edition: Optional[str] = None
+    abstract: Optional[str] = None
+    publisher: Optional[str] = None
     catalog: Optional[str] = None
     category_type: Optional[str] = None
     category_age: Optional[str] = None
     category_topics: Optional[str] = None
     langage: Optional[str] = None
-    cover: Optional[str] = None
+    cover_url: Optional[str] = None
```
Comment (carole-j) [inline backend/models.py:86]: any renaming of class's arguments should impact unit tests

File: backend/main.py [hunk 2]
```diff
@@ -15,7 +17,48 @@ async def startup(app: FastAPI):
...
-app.include_router(member.router)
-app.include_router(circulation.router)
+API_PREFIX = "/api/v1"
+
+app.include_router(book.router, prefix=API_PREFIX)
+app.include_router(family.router, prefix=API_PREFIX)
+app.include_router(member.router, prefix=API_PREFIX)
+app.include_router(circulation.router, prefix=API_PREFIX)
+
+# Auth
```
Comment (carole-j) [inline backend/main.py:27]: Do you think a file for the auth part could be useful?

File: backend/models.py [hunk 3]
```diff
@@ -24,15 +46,18 @@ class SharedUpdate(SQLModel):
...
-    edition: Optional[str] = None
+    abstract: Optional[str] = None
+    publisher: Optional[str] = None
     catalog: Optional[str] = Field(default=None, index=True)
     category_type: Optional[str] = Field(default=None, index=True)
     category_age: Optional[str] = Field(default=None, index=True)
     category_topics: Optional[str] = None
-    langage: Optional[str] = None
-    cover: Optional[str] = None
+    langage: Optional[LanguageAlpha2] = None
```
Comment (carole-j) [inline backend/models.py:55]: which writing should we take: langage or language?

File: backend/models.py (hunk 1 above)
Comment (denis-jullien) [reply backend/models.py]: ok

File: backend/models.py (hunk 3 above)
Comment (denis-jullien) [reply backend/models.py]: language is correct

**Pull request merged**